MonstieWash/Assets/Scripts/UI/ScoreUIManager.cs

```diff
@@ -0,0 +1,137 @@
+using JetBrains.Annotations;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using TMPro;
+using Unity.VisualScripting;
+using UnityEngine;
+using UnityEngine.SocialPlatforms.Impl;
+using UnityEngine.UI;
+
+public struct PHTask
+{
+    public string name;
+    public bool complete;
+    public int score;
+
+    public PHTask(string _name, bool _complete, int _score)
+    {  name = _name; complete = _complete; score = _score; }
```

Comment:
I think a more typical C# way of doing this (pretty sure it's the microsoft recommended convention) is to use `this.` for this kinda stuff so you don't need the underscores. I used to do it like this too until I got told off for it haha
```suggestion
    public PHTask(string name, bool complete, int score)
    {  this.name = name; this.complete = complete; this.score = score; }
```